src/Fullpage.tsx

```diff
@@ -226,10 +231,16 @@ export default function Fullpage({
   const goto = (newSlide, resetScroll = false) => {
     const { transitionTiming, fullpageHeight, viewportHeight } = pageState;
 
-    if (slide !== newSlide && newSlide) {
+    console.info('\n\n\n in goto', slide, newSlide, slide != newSlide);
+
+    // TODO(noah): this is always false
```

Comment:
this is the current issue, which likely is a consequence of:
- moving to slide 1 (state is now 1)
- then immediately returning to slide 0 (but state is still 1)
- now when trying to move next (state says 1 != 1) so dont do anything

Reply:
instead of trying to keep the existing code

it may be easier to just trash it all and use it for inspiration of something new

the code is wack anway